frappe/www/login.py

```diff
@@ -11,13 +11,7 @@
 from frappe.utils import cint, get_url
 from frappe.utils.html_utils import get_icon_html
 from frappe.utils.jinja import guess_is_path
-from frappe.utils.oauth import (
-	get_oauth2_authorize_url,
-	get_oauth_keys,
-	login_via_oauth2,
-	login_via_oauth2_id_token,
-	redirect_post_login,
-)
+from frappe.utils.oauth import get_oauth2_authorize_url, get_oauth_keys, redirect_post_login
```

Comment:
This change is a tad bit concerning but IMO should be fine - given the utils are actually just being redefined here...